arduino-sht/README.md

```diff
@@ -0,0 +1,131 @@
+# arduino-sht
```

Comment:
I'm not quite sure why this is part of the diff.. the file existed before, right?